docs/deployment-instructions.md

```diff
@@ -0,0 +1,164 @@
+# z/OS Deployment Instructions
+
+There are multiple approaches to deploying the sample API service to z/OS; the following
+serves as a reference to deploy the sample API as-is.  For a production instance of an API, your deployment process may differ.
+
+**(Substitute values below for your site configuration)**
+
+## Manual Deployment
+
+First, create a space to deploy the API artifacts.
+
+Login to [z/OS Unix Shell](https://www.ibm.com/support/knowledgecenter/zosbasics/com.ibm.zos.zconcepts/zconcepts_146.htm).
+
+**(You may need privileged authority to issue the command examples below)**
+
+1. Allocate and format a [z/OS File System](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.bpxb200/zfspref.htm) (zFS):
+
+    - `zfsadm define -aggregate IBMUSER.SAMPLAPI.ZFS -cyls 100 -volumes WRKD23`
+      - (response): `IOEZ00248I VSAM linear dataset IBMUSER.SAMPLAPI.ZFS successfully created.`
+    - `zfsadm format -aggregate IBMUSER.SAMPLAPI.ZFS`
+      - (response): `IOEZ00077I HFS-compatibility aggregate IBMUSER.SAMPLAPI.ZFS has been successfully created`
+
+2. Create directory and mount the file system:
+
+    - `mkdir /u/ibmuser/samplapi`
+    - `/usr/sbin/mount -v -f IBMUSER.SAMPLAPI.ZFS /u/ibmuser/samplapi`
+      - response: `FOMF0502I Mount complete for IBMUSER.SAMPLAPI.ZFS`
+
+### Deploy Artifacts
+
+Next, deploy artifacts from your workstation to the zFS.
+
+You can upload artifacts via `ftp`, `sftp`, `scp` or [`Zowe CLI`](https://github.com/zowe/zowe-cli). `zowe` commands will be used in the remaining examples.
+
+#### Deploy the Sample Service API Jar
+
+To obtain the sample service jar, run `gradlew build`.  The default artifact will be `build/libs/zowe-apiservice-0.0.1-SNAPSHOT.jar`.
+
+1. Create a directory for the `sample-service.jar`
+   - `mkdir /u/ibmuser/samplapi/jars`
+
+2. Upload the `sample-service.jar` as a binary artifact:
+
+    - `zowe files upload ftu "<path_to_local_file>/sample-service.jar" "/u/ibmuser/samplapi/jars/sample-service.jar" --binary`
+
+#### Deploy the Sample Service Configuration YAML
+
+1. Create a directory for the `application.yml`
+   - `mkdir /u/ibmuser/samplapi/config`
+
+2. Upload the `config/local/application.yml` as a binary artifact:
+
+`zowe files upload ftu "config/local/application.yml" "/u/ibmuser/samplapi/config/local/application.yml" --binary`
```

Comment:
I would remove `local` from `/u/ibmuser/samplapi/config/local/application.yml`. The `local` meant configuration for running on a *local* workstation.